### PR TITLE
#9 seed database

### PR DESCRIPTION
**Pull request by Fahi28**
All required tables have been seeded. Have a look at the regions table in case there are any region names you aren't happy with.

### CODE REVIEW

**Comment by S1mpySloth** [comment]
Previous conversation regarding grid-sizes (has since been addressed)

"The finer, the better right? So users can subscribe to as-specific of a region as possible
What’s the trade off? In refining it into giving us more square grids?"

Trade-off was that the naming-convetions may be tedious to type out at 144 grids (compared to the 36 we have now)
and it may also have been confusing to present users with 144 dropdown options to subscribe to.

**Review by qrafiq1 — CHANGES_REQUESTED** [review]
Seeding looks good, one thing I noticed is region_id can be SMALLINT isntead of BIGINT

File: schema.sql [hunk 1]
```diff
@@ -86,4 +86,82 @@ CREATE TABLE user_topic_assignment (
...
+    ('se'), 
+    ('us'), 
+    ('uu'), 
+    ('uw');
+
+INSERT INTO type (type_name) VALUES 
+    ('earthquake'), 
+    ('quarry');
+
+INSERT INTO regions (min_latitude, max_latitude, min_longitude, max_longitude, region_name) VALUES
```
[the remark this answers is not in the document]
Comment (Fahi28) [reply schema.sql:126]: region_id is auto incrementing so there is no need to include it. regarding the example mentioned, I believe we talking about having the lower bound of each region being inclusive and the upper bound being exclusive.

**Review by jiuliangut — APPROVED** [review]
Nice - good idea adding constraints to check lat and long

**Pull request merged**